themes/gatsby-theme-blog/README.md

```diff
@@ -0,0 +1,52 @@
+<p align="center">
+  <a href="https://www.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  The Gatsby blog theme
+</h1>
+
+A Gatsby theme for creating a blog.
+
+## Installation
+
+### Use the blog theme starter
+
+This will generate a new site that pre-configures use of the blog theme.
+
+```sh
+gatsby new my-themed-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
+```
+
+### Manually add to your site
+
+```sh
+yarn add gatsby-theme-blog
+```
+
+## Usage
+
+```js
+// gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-theme-blog`,
+      options: {
+        // basePath defaults to `/`
+        basePath: `/blog`,
+      },
+    },
+  ],
+}
+```
+
+### Options
```

Comment:
Where are we putting docs for the `social`, etc in the `siteMetadata`?

Reply:
Good point. I was just thinking actual theme options. The siteMetadata options are pseudo-documented via the blog starter. I should put it in here, too.

Reply:
Added.